### PR TITLE
*: enable gosec

### PR DESCRIPTION
**Pull request by hawkingrei**
Signed-off-by: Weizhen Wang <wangweizhen@pingcap.com>

### What problem does this PR solve?

Issue Number: ref #35345

Problem Summary:

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: br/pkg/lightning/config/bytesize_test.go [hunk 1]
```diff
@@ -89,11 +89,11 @@ func TestByteSizeTOMLDecode(t *testing.T) {
 		},
 		{
 			input: "x = ['100000']",
-			err:   "toml: cannot load TOML value.*",
+			err:   "toml: incompatible types:.*",
```
Comment (hawkingrei) [inline br/pkg/lightning/config/bytesize_test.go:92]: Some linter needs to upgrade the version of toml. so it change some error message.

**Review by tangenta — APPROVED** [review]

**Review by tisonkun — APPROVED** [review]

**Pull request merged**